### PR TITLE
Operations rubric

### PR DESCRIPTION
**Pull request by dinesh-skyach**
Operations rubric

### CODE REVIEW

File: app/controllers/admin/balance_sheet_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# encoding: UTF-8
+# frozen_string_literal: true
+
+module Admin
+  class BalanceSheetController < BaseController
+    def index
+      @assets = ::Operations::Asset.group(:currency_id).sum(:credit)
```
Comment (ysv) [inline app/controllers/admin/balance_sheet_controller.rb]: @dinesh-skyach asset balance is difference between sum of credits and debits 
`sum(:credit) - sum(:debit)`

**Review by ysv — CHANGES_REQUESTED** [review]

**Comment by mod** [comment]
1- Do not nest Currencies inside Operations types
2- show all operations by default with an ability to filter with a dropdown currency selector
3- balance sheet is mistaken, liabilities must be equal to member balances.

File: app/controllers/admin/operations/assets_controller.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# encoding: UTF-8
+# frozen_string_literal: true
+
+require_dependency 'admin/operations/base_controller'
+
+module Admin
+  module Operations
+    class AssetsController < BaseController
+      def index
+        @assets = ::Operations::Asset.includes(:reference, :currency)
```
Comment (ysv) [inline app/controllers/admin/operations/assets_controller.rb:10]: Do we need includes here ?

File: app/views/admin/operations/shared/_table.html.erb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+
+<%= table_for objects, class: 'table table-bordered table-striped' do |t| %>
+  <% t.column 'Currency' do |x| %>
+    <% x.currency.code.upcase %>
```
Comment (ysv) [inline app/views/admin/operations/shared/_table.html.erb:4]: x.currency_id

File: app/controllers/admin/balance_sheet_controller.rb (hunk 1 above)
Comment (dinesh-skyach) [reply app/controllers/admin/balance_sheet_controller.rb]: @ysv Done

**Pull request merged**